configure.py

```diff
@@ -102,6 +102,9 @@ def generate_shared_lib_name(namespec):
 with open("requirements.txt") as f:
     _REQUIRED_PKG = f.read().splitlines()
 
+with open("build_deps/build-requirements.txt") as f:
+    _REQUIRED_PKG.extend(f.read().splitlines())
+
```

Comment:
Can you add a comment explaining why we don't use the "-r" option of pip? Thanks!

Reply:
Sure, the reason is just so we can print the dependencies in L116. I can add a comment for that, but wanted to make sure you don't see a better method. 

Currently `check_deps` does not return what package is missing (and will require a refactor since its called 2x and can exit on the first one. The current setup is not ideal, but I'm not really sure its worth the trouble to change it.

Reply:
Yeah, this PR is important. I don't want to block it for something like that. Let's just remove the exit(0) and I'll merge it.

Reply:
Done. Thanks